src/main/java/juja/microservices/teams/controller/TeamController.java

```diff
@@ -11,7 +11,11 @@
 import javax.inject.Inject;
 import javax.validation.Valid;
 
+/**
+ * @author Ivan Shapovalov
+ */
 @RestController
+@RequestMapping(value = "/v1")
```

Comment:
You can set request mapping for whole class - /v1/teams, and remove `/teams` from methods mapping.